src/main/ycsb/ycsb_workload.cpp

```diff
@@ -84,16 +84,20 @@ volatile bool is_running = true;
 PadInt *abort_counts;
 PadInt *commit_counts;
 
+#ifndef __APPLE__
 void PinToCore(size_t core) {
   cpu_set_t cpuset;
   CPU_ZERO(&cpuset);
   CPU_SET(core, &cpuset);
   pthread_setaffinity_np(pthread_self(), sizeof(cpu_set_t), &cpuset);
 }
+#endif
 
 void RunBackend(const size_t thread_id) {
-  
+
+#ifndef __APPLE__
   PinToCore(thread_id);
+#endif
```

Comment:
Refer to previous comment. Leave `PinToCore` in, but pound-define out implementation. It'll be compiled away on OSX.